study.md

```diff
@@ -21,6 +21,11 @@ Ember](https://www.codeschool.com/courses/try-ember) course from CodeSchool.
 
 In your own words, describe why someone would use Ember.
 
-```md
-<!-- your answer here -->
-```
+You would use Ember.js framework to build a fast-loading interactive single-page web app. The framework is fairly easy to get started in. A lot of the Model-View-Controller files and handlebars templates are generated for you via command line scripts much like Rails.
+
+Ember.js apps loads the full website in the initial network request. The only requests the user makes from the server is data. Other requests after from the user to the website are between the client browser and the user. In other words, server serves the data and client serves the user.
```

Comment:
Yes, Ember loads everything.  Sometimes it even has some of the data, using Ember-data.